### ISSUE TITLE
I want to access my channel management for a token from the pay screen

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#321]
## User Story
As Chris, I want to have a simple way to manage my channels regardless of their state, so that I can maintain them with low effort and no confusion.

*Should the layout of the different states  (open, closed, settleable) be determined by the channel status, with interfaces that change according to it?*

## Acceptance criteria
- User intuitively knows the status of the channel just by looking at the layout (consider factors like positioning of elements, color coding and/or icons).

## Tasks
- [ ] Design
 - [ ] Implementation

## Remarks
- Looking how it works from a UX perspective in Lightning

**Comment by taleldayekh** [comment]
If we break down the concept of channels into actions that a users might want to perform when maintaining their channels, these would be:

1. Deposit tokens into a channel
2. Close/settle a channel

This user journey illustrates both action 1 and action 2:
![maintaining channels](https://user-images.githubusercontent.com/43838780/66833218-b63ca180-ef5b-11e9-937b-c4aca997ae27.jpg)
---
The most commonly performed action would be **deposit**, where users can top up their channels before making a payment/transfer. It therefore makes sense for this action to be easily accessible for the user.

In this new flow, an option to deposit has been added to a "main menu". This wireframe illustrates how depositing into a channel would look like:
![Issue #321](https://user-images.githubusercontent.com/43838780/66831590-51337c80-ef58-11e9-9ec3-c07f1f53d1de.jpg)

The second action of **closing and settling** a channel will be moved to a separate **settings** screen. This is because a user presumably closes channel's less often than making deposits.

**Comment by christianbrb** [comment]
@taleldayekh @roy-castrillo Could you rephrase the issue so that it reflects our discussion from the refinement? Thanks : )

**Comment by taleldayekh** [comment]
Done

**Comment by christianbrb** [comment]
We need to decide if #433 fixes it or not

**Comment by christianbrb** [comment]
Closing this as we are heading into a different direction
